src/server/services/collection.service.ts

```diff
@@ -1025,6 +1026,10 @@ export const getCollectionItemsByCollectionId = async ({
     return [];
   }
 
+  if (user && forReview) {
+    user.isModerator = true;
+  }
```

Comment:
Are we sure this won't bring us any issues down the line?

Reply:
All things considered I don't think so. The only downside is leaking some images (non-NSFW ones) and I believe that's the goal with what Matty wants. it's not ideal for sure, but the other option (which is a cross-query between images<->posts<->collectionContributors is really not an option performance wise.